src/Main.elm

```diff
@@ -76,3 +202,40 @@ fetchPerform : String -> Cmd Msg
 fetchPerform url =
   Task.perform FetchFail FetchSuccess (Http.get decoder url)
 
+wheelEventDecoder : Json.Decoder WheelEvent
+wheelEventDecoder =
+  Json.object3 WheelEvent
+    ("deltaX" := Json.float)
+    ("deltaY" := Json.float)
+    ("deltaZ" := Json.float)
+
+movePolygonPosition : Model -> List ( List ( Float, Float ) )
+movePolygonPosition model =
```

Comment:
I think this could be cleaner, see https://github.com/trifectalabs/elm-maps/blob/4923a2416ef0b0c17836b05168c41fb432fe8015/src/GeoJsonHelpers.elm#L80